package/MDAnalysis/coordinates/DCD.py

```diff
@@ -522,26 +535,36 @@ def timeseries(self, asel, start=0, stop=-1, skip=1, format='afc'):
         # Check if the atom numbers can be grouped for efficiency, then we can read partial buffers
         # from trajectory file instead of an entire timestep
         # XXX needs to be implemented
-        return self._read_timeseries(atom_numbers, start, stop, skip, format)
+        return self._read_timeseries(atom_numbers, start, stop, step, format)
 
-    def correl(self, timeseries, start=0, stop=-1, skip=1):
+    def correl(self, timeseries, start=None, stop=None, step=None, skip=None):
```

Comment:
> On 30 Jul, 2016, at 14:51, John Detlefs <notifications@github.com> wrote:
> 
> Alright. This PR was a bit of a pandora's box I guess 😃

More like a hornets' nest.

But yes, you're right. Much more than "yeah, just fix the docs".

--
Oliver Beckstein * orbeckst@gmx.net
skype: orbeckst  * orbeckst@gmail.com